src/components/sidebar.scss

```diff
@@ -132,12 +132,14 @@
     @include sidebar-drawer-global;
   }
 
+  // --op-breakpoint-medium
```

Comment:
This can move down a line to be directly above the media query instead of above the whole & selector.